xcode.mk

```diff
@@ -38,14 +47,15 @@ release-with-icu:
 	-DCMAKE_INSTALL_PREFIX="$(RIME_DIST_DIR)" \
 	-DCMAKE_PREFIX_PATH="$(ICU_PREFIX)" \
 	$(CMAKE_BOOST_OPTIONS) \
-	$(CMAKE_OSX_ARCHS)
+	$(CMAKE_OSX_OPTIONS)
 	cmake --build $(build) --config Release
 
 debug:
 	cmake . -B$(build) -GXcode \
 	-DBUILD_STATIC=ON \
 	-DBUILD_SEPARATE_LIBS=ON \
-	$(CMAKE_BOOST_OPTIONS)
+	$(CMAKE_BOOST_OPTIONS) \
+	$(CMAKE_OSX_OPTIONS)
```

Comment:
Nice to have this change.
Initially I ommited the architecture option for the debug build because I found it is much slower than a native build, not good for fast iteration. Now this is a not an issue any more thanks to the `BUILD_UNIVERSAL` variable.